examples/dynamic-bridge-app/linux/main.cpp

```diff
@@ -176,6 +189,141 @@ std::unique_ptr<GeneratedCluster> CreateCluster(chip::ClusterId id)
     return nullptr;
 }
 
+void ReadValueFromBuffer(chip::TLV::TLVWriter & wr, const char * buffer, const Span<const char> *)
+{
+    wr.PutString(chip::TLV::Tag(), buffer, (uint32_t) strlen(buffer));
+}
+void ReadValueFromBuffer(chip::TLV::TLVWriter & wr, const char * buffer, const ByteSpan *)
+{
+    wr.PutBytes(chip::TLV::Tag(), (const uint8_t *) buffer, (uint32_t) strlen(buffer));
+}
+void ReadValueFromBuffer(chip::TLV::TLVWriter & wr, const char * buffer, const float *)
+{
+    wr.Put(chip::TLV::Tag(), (float) atof(buffer));
```

Comment:
is this and below correct? are we really writing string data into buffers?

What is the path that gets data into buffer? I thought ember internals are generally binary data with endianess taken into account.

Reply:
Addressed in [42661f6](https://github.com/project-chip/connectedhomeip/pull/23840/commits/42661f626408834ef130441ce658c9b021ff1e02)